janitor/functions/conditional_join.py

```diff
@@ -65,38 +64,58 @@ def conditional_join(
     a single index column is returned; else, a MultiIndex column
     is returned.
 
-    Functional usage syntax:
-
-    ```python
-        import pandas as pd
-        import janitor as jn
-
-        df = pd.DataFrame(...)
-        right = pd.DataFrame(...)
-
-        df = jn.conditional_join(
-                df,
-                right,
-                (col_from_df, col_from_right, join_operator),
-                (col_from_df, col_from_right, join_operator),
-                ...,
-                how = 'inner' # or left/right
-                sort_by_appearance = True # or False
-                )
-    ```
-
-    Method chaining syntax:
-
-    ```python
-        df.conditional_join(
-            right,
-            (col_from_df, col_from_right, join_operator),
-            (col_from_df, col_from_right, join_operator),
-            ...,
-            how = 'inner' # or left/right
-            sort_by_appearance = True # or False
-            )
-    ```
+    Example:
+
+        >>> import pandas as pd
+        >>> import janitor
+        >>> df1 = pd.DataFrame({"value_1": [2, 5, 7, 1, 3, 4]})
+
+        >>> df2 = pd.DataFrame({"value_2A": [0, 3, 7, 12, 0, 2, 3, 1],
+        ...                     "value_2B": [1, 5, 9, 15, 1, 4, 6, 3],
+        ...                    })
+
+        >>> df1
+           value_1
+        0        2
+        1        5
+        2        7
+        3        1
+        4        3
+        5        4
+
+        >>> df2
+           value_2A  value_2B
+        0         0         1
+        1         3         5
+        2         7         9
+        3        12        15
+        4         0         1
+        5         2         4
+        6         3         6
+        7         1         3
+
+
+        >>> df1.conditional_join(
+        ...     df2,
+        ...     ("value_1", "value_2A", ">="),
+        ...     ("value_1", "value_2B", "<=")
+        ... )
+            value_1  value_2A  value_2B
+        0         2         1         3
+        1         2         2         4
+        2         5         3         5
+        3         5         3         6
+        4         7         7         9
+        5         1         0         1
+        6         1         0         1
+        7         1         1         3
+        8         3         1         3
+        9         3         2         4
+        10        3         3         5
+        11        3         3         6
+        12        4         2         4
+        13        4         3         5
+        14        4         3         6
 
 
     :param df: A Pandas DataFrame.
```

Comment:
Some comments pertaining to the docstring of the params:

1. `df`
```suggestion
    :param df: A pandas DataFrame.
```

2. `conditions`: Might it be worth mentioning that "if multiple conditions are passed, they will be combined together using a `logical_and`"? (if you have some better way of phrasing this, pls do!)